.github/workflows/test-aks-deploy.yaml

```
@@ -0,0 +1,95 @@
+  BUILD_CONTEXT_PATH: ./
+  NAMESPACE: default
+
+jobs:
+  buildImage:
+    permissions:
+      contents: read
+      id-token: write
+    runs-on: ubuntu-latest
+    steps:
+      # Checks out the repository this file is in
+      - uses: actions/checkout@v3
```

actionlint (1.7.10)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)